### ISSUE TITLE
Release version stucks in `virt_points_phys_recur`

### ISSUE THREAD

**Issue by toku-sa-n** [toku-sa-n/ramen#45]

**Comment by toku-sa-n** [comment]
When `table == TableType::Pdpt`, it stucks.

```rust
fn virt_points_phys_recur(
    virt: usize,
    phys: usize,
    table_addr: usize,
    mem_map: &mut [boot::MemoryDescriptor],
    table: TableType,
) -> () {
    stop_if!(table == TableType::Pdpt); // Not stucks.
    stop_if!(table == TableType::Pd); // Stucks before this code.
```

**Comment by toku-sa-n** [comment]
https://github.com/toku-sa-n/ramen/blob/6b10fdff94bcf1e67fc69dd390d3dc4325b7367e/bootx64/src/memory.rs#L132

In this line booting stucks.

**Comment by toku-sa-n** [comment]
https://github.com/toku-sa-n/ramen/blob/6b10fdff94bcf1e67fc69dd390d3dc4325b7367e/bootx64/src/memory.rs#L188

This is the cause, but why does it reach here?

**Comment by toku-sa-n** [comment]
https://github.com/toku-sa-n/ramen/blob/c977a938899113583e3393edb46dc9525b8f5a59/Makefile#L36
Changing `-m 4G` to `-m 8G` solved the problem, but it doesn't solve the actual problem.

**Comment by toku-sa-n** [comment]
Maybe this hang is because the size of `MemoryDescriptor` is not the same as `size_of::<MemoryDescriptor>()`?

**Comment by toku-sa-n** [comment]
Fixed by #57
